documentation/api/wire_format/report_format_v5.markdown

```diff
@@ -99,9 +99,8 @@ match a report with the catalog that was used for the run.  This field may be `n
       "time" : <log line timestamp>
      }
 
-**Note on fields that allow `NULL` values**
-
-In the resource_event schema above, `containment_path`, `new_value`, `old_value`, `property`, `file`, `line`, `status`, and `message` may all be null.
+>**Note: Fields that allow `null` values**
+>In the resource_event schema above, `containment_path`, `new_value`, `old_value`, >`property`, `file`, `line`, `status`, and `message` may all be null.
```

Comment:
there's a > before property here. Doesn't render right in GH, not sure what it does on the site. It'd be clearer on a separate line.